service/src/main/java/bio/terra/workspace/app/configuration/spring/BeanConfig.java

```diff
@@ -22,13 +24,49 @@ public NamedParameterJdbcTemplate getNamedParameterJdbcTemplate(
     return new NamedParameterJdbcTemplate(config.getDataSource());
   }
 
+  public static class HTMLCharacterEscapes extends CharacterEscapes {
+    private final int[] asciiEscapes;
+
+    public HTMLCharacterEscapes() {
+      // Start with a copy of the default set of escaped characters, then modify.
+      int[] esc = CharacterEscapes.standardAsciiEscapesForJSON();
+      // Escape HTML metacharacters for security reasons. For JSON, CharacterEscapes.ESCAPE_STANDARD
+      // means unicode-escaping.
+      esc['<'] = CharacterEscapes.ESCAPE_STANDARD;
+      esc['>'] = CharacterEscapes.ESCAPE_STANDARD;
+      esc['&'] = CharacterEscapes.ESCAPE_STANDARD;
+      asciiEscapes = esc;
+    }
+
+    /** Return the escape codes used for ASCII characters. */
+    @Override
+    public int[] getEscapeCodesForAscii() {
+      return asciiEscapes;
+    }
+
+    /**
+     * Return the escape codes used for non-ASCII characters, or ASCII characters with escape code
+     * set to ESCAPE_CUSTOM.
+     *
+     * <p>This is required because CharacterEscapes is abstract. We don't need additional escaping
+     * for any characters, so per documentation this can return null for all inputs.
+     */
+    @Override
+    public SerializableString getEscapeSequence(int ch) {
+      return null;
```

Comment:
should this actually return something?

Reply:
We don't need any additional custom escape behavior (either escaping non-ASCII characters or ASCII characters set to `CharacterEscapes.ESCAPE_CUSTOM` above), so this should return `null` for all inputs. Per superclass documentation:
`Returns:
Escape sequence to use for the character, if any; null if not`

(but agreed, it looks kind of odd)